src/lib/components/progressBar/progressBar.js

```diff
@@ -13,21 +13,21 @@ const ProgressBar = ({
   height,
   fontSize,
 }) => {
-  const progressBarStyle = {
+  const progressStyle = {
     width: `${progress}%`,
     fontSize: `${fontSize}px`,
     lineHeight: `${height}px`,
   };
-  const progressStyle = {
+  const progressBarStyle = {
     height: `${height}px`,
   };
-  if (color) progressBarStyle.color = color;
-  if (barColor) progressBarStyle.backgroundColor = barColor;
-  if (backgroundColor) progressStyle.backgroundColor = backgroundColor;
+  if (color) progressStyle.color = color;
```

Comment:
Does this work also? If it does it will reduce the amount of code: `progressStyle.color = color && color;` Or
```
 const progressBarStyle = {
    height: `${height}px`,
    color: color && color
  };
```

Reply:
hmm I'm a bit confused @bronz3beard - we will need the other properties too as they're defined up above. I'm gonna merge in interest of getting it into MR but let's come back to this with @davidpaley :)

Reply:
@bronz3beard it is a good idea what you suggested but it is not correct to put `color && color`. In this case you should put `color || undefined`. You should define everything like this:
```
 const progressBarStyle = {
    backgroundColor: barColor || backgroundColor  || undefined,
    color: color || undefined
  };
```

Reply:
hey @bronz3beard if you want to open a new pr to refactor this, let me know :)

Reply:
I think the best solution better than `undefined` is to give it a `defaultProp`, but it really depends what its used for i guess, as i wrote this i noticed its already merged but im this far so ill just leave this here 😅